metrics/storage/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Set up an Ubuntu image with 'blogbench' installed
+
+# Usage: FROM [image name]
+FROM ubuntu
```

Comment:
@GabyCT , can you investigate for me if/how we might bury a version in the Dockerfile, and then adjust our `docker build` check/invocation so we can spot if the dockerfile has been updated, and rebuild the image. Right now the (rather simple) code I wrote only does a naive check to see if the image already exists, and does not have any capability to update the image. I think one day that will come to bite us....